airbyte-integrations/bases/source-acceptance-test/unit_tests/test_spec_unit.py

```diff
@@ -0,0 +1,101 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+import io
+
+import docker
+import pytest
+from source_acceptance_test.utils import ConnectorRunner
+
+
+def build_docker_image(text: str, tag: str) -> docker.models.images.Image:
+    """
+    Really for this test we dont need to remove the image since we access it by a string name
+    and remove it also by a string name. But maybe we wanna use it somewhere
+    """
+    client = docker.from_env()
+    fileobj = io.BytesIO(bytes(text))
+    image, iterools_tee = client.images.build(fileobj=fileobj, tag=tag, forcerm=True, rm=True)
+    return image
+
+
+@pytest.fixture
+def correct_connector_image() -> str:
+    dockerfile_text = b"""
```

Comment:
```suggestion
    dockerfile_text = """
```

Reply:
Dockerfile failing with exception and asking to use BytesIo instead of StringIO.  BytesIO this type of file-like object required binary content (or encoding provided to encode the text), so I have to use binary strings in this cases. This case will not work if I pass the normal text